src/test/testing/display/picker.unit.test.ts

```diff
@@ -99,3 +108,78 @@ suite('Unit Tests - Picker (execution of commands)', () => {
         });
     });
 });
+
+suite('Testing - TestDisplay', () => {
+
+    const wkspace = Uri.file(__dirname);
+    let mockedCommandManager: ICommandManager;
+    let mockedServiceContainer: IServiceContainer;
+    let mockedTestCollectionStorage: ITestCollectionStorageService;
+    let mockedAppShell: IApplicationShell;
+    let mockedFileSytem: IFileSystem;
+    let testDisplay: ITestDisplay;
```

Comment:
FWIW, since we are testing an actual `TestDisplay`, it can be slightly more clear to the reader if you specify the class rather than the interface.

```suggestion
    let testDisplay: TestDisplay;
```